iac/stacks/constructs/worker.py

```diff
@@ -53,9 +53,9 @@ def __init__(
             # SQS queue's ApproximateNumberOfMessagesVisible metric
             scaling_steps=[
                 # when <=10 messages, scale down
-                appscaling.ScalingInterval(upper=10, change=-1),
+                appscaling.ScalingInterval(upper=100, change=-1),
```

Comment:
Just checking this is desirable for all workers?

Reply:
That was one of the changes I made with Mike, to reduce when it would need to autoscale since we don't have the longer running tasks anymore with update_summary.